.github/workflows/test.yaml

```diff
@@ -10,15 +10,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.8', '3.11']
+        python-version: ['3.9', '3.11']
```

Comment:
```suggestion
        python-version: ['3.9', '3.13']
```

Reply:
This one was deliberate, if there aren't wheels for all of Galaxy's packages then it takes a long to build them to run the tests, and they don't always exist for the combo of older Galaxy release and newer version of Python that we test.

That said we could just only test the old release on 3.9, and dev on 3.9 and 3.13. We were already doing that before.

Reply:
Let's see if what I said even applies for 3.13 + release_23.0...

Reply:
```
Ignoring a2wsgi: markers 'python_version >= "3.7" and python_version < "3.12"' don't match your environment
...
```

Well, indeed. :laughing: